packages/client/components/ReviewRequestToJoinOrgModal.tsx

```diff
@@ -42,24 +62,39 @@ interface Props {
 const ReviewRequestToJoinOrgModal = (props: Props) => {
   const {closePortal, queryRef} = props
 
+  const atmosphere = useAtmosphere()
+
   const data = usePreloadedQuery<ReviewRequestToJoinOrgModalQuery>(query, queryRef)
 
   const [selectedTeams, setSelectedTeams] = useState<string[]>([])
 
   const {domainJoinRequest} = data.viewer
 
+  useEffect(() => {
+    if (domainJoinRequest) {
+      SendClientSegmentEventMutation(atmosphere, 'Join Request Reviewed')
```

Comment:
+1 Would change this name:
```suggestion
      SendClientSegmentEventMutation(atmosphere, 'Join Request Viewed')
```

It also might make sense to move this out of the modal logic, and instead emit the event when they see the Join Request notification.  That way the funnel we'd be tracking is "Join Request Viewed" (emitted for every user who receives a request when they view it) -> "Join Request Reviewed" when they either "Deny" or add the user to teams.  I'm not sure we learn a lot from people who are just opening the modal.

Reply:
We have existing `Snackbar Viewed` event, should we use it when snackbar with the request viewed?

Reply:
Made some changes